projects/angular-cld/src/lib/analyticsOptionsDefault.ts

```diff
@@ -0,0 +1,8 @@
+import {VERSION} from '@angular/core';
+import {APP_VERSION} from './version';
+
+export let analyticsOptionsDefault = {
```

Comment:
Any reason not to use "defaultAnalyticsOptions"? 

It's not much about the order, but the fact "defaultAnalyticsOptions" is a valid English phrase while analyticsOptionsDefault is not.

Reply:
After reading further:

Can this value change? 
If it can't change, I wouldn't call it "default" (as default implies it might be overwritten with some new data)

SDKAnalyticsConstants or something with that flavor might be more suitable